src/test/java/pl/jakubkozlowski/learning/firststeps/service/ItemPageServiceImplTest.java

```diff
@@ -74,47 +84,22 @@ public void whenSave_thenReturnItemPageDTO() {
                 .isEqualTo(itemPageDTOAttack);
     }
 
-    @Test
-    public void whenFindAll_thenReturnListOfItemPagesDTO() {
-        //when
-        List<ItemPageDTO> actual = itemPageService.findAll();
-        //then
-        assertThat(actual)
-                .isEqualTo(itemPageDTOList);
-    }
-
     @Test
     public void whenFindById_thenReturnItemPageDTO() {
         //when
-        ItemPageDTO actual = itemPageService.findById(itemPageEntityAttack.getId());
+        Optional<ItemPageDTO> actual = itemPageService.findById(itemPageEntityAttack.getId());
         //then
         assertThat(actual)
-                .isEqualTo(itemPageDTOAttack);
+                .isEqualTo(Optional.of(itemPageDTOAttack));
     }
 
     @Test
-    public void whenFindByName_thenReturnUncompletedItemPageDTO() {
+    public void whenFindByPagename_thenReturnUncompletedItemPageDTO() {
         //when
-        ItemPageDTO actual = itemPageService.findByPagename(itemPageEntityAttack.getPagename());
+        Optional<ItemPageDTO> actual = itemPageService.findByPagename(itemPageEntityAttack.getPagename());
```

Comment:
Do not wrap into optional for testing. That is not needed and redundant. Look here for optional: http://joel-costigliola.github.io/assertj/assertj-core-news.html